tests/unit/test_game.py

```diff
@@ -470,6 +470,41 @@ def test_fictitious_play(self, A, B, seed):
             assert np.array_equal(column_play, expected_column_play)
         # assert expected_outcome == outcome
 
+    # @given(
+    #    A=arrays(np.int8, (4, 3)),
+    #    B=arrays(np.int8, (4, 3)),
+    #    seed=integers(min_value=0, max_value=2 ** 32 - 1),
+    # )
```

Comment:
If this is not needed, let us remove it

```suggestion
```

Reply:
This is here for discussion in our meeting! It should be there but I will have to make a couple of changes to get it to work properly (not sure what these are yet)! But understand why we shouldn't 'save' code by commenting it because of version control.

Reply:
No that's fine, for the purpose of a discussion that's absolutely fine. Next time add:

```
# TODO discuss this in meeting:
```